src/Pipeline.h

```diff
@@ -184,11 +184,17 @@ class Pipeline {
      * outputs. Schedules the Funcs compute_root(). */
     Pipeline(const std::vector<Func> &outputs);
 
+    /** Make a pipeline from deserialization. */
```

Comment:
This comment is confusing, because the args are not a "deserialization". A better comment would be something like: "Make a Pipeline with a vector of outputs and a list of requirement assertions. Used by deserialization."